src/v2/styles/MessageActionsBox/MessageActionsBox-layout.scss

```diff
@@ -21,22 +21,11 @@
 
 .str-chat__message-actions-box:not(.str-chat__message-actions-box-angular) {
   display: none;
-  position: absolute;
```

Comment:
`react-popper` will take care of positioning, so none of this is required anymore.